pymc3/distributions/transforms.py

```diff
@@ -510,6 +513,62 @@ def t_stick_breaking(eps):
     return StickBreaking(eps)
 
 
+class StickBreaking2(Transform):
+    """
+    Transforms K - 1 dimensional simplex space (k values in [0,1] and that sum to 1) to a K - 1 vector of real values.
+    """
+
+    name = "stickbreaking2"
+
+    def __init__(self, eps=None):
+        if eps is not None:
+            warnings.warn("The argument `eps` is depricated and will not be used.",
+                          DeprecationWarning)
```

Comment:
I did this because I replaced the original stick-breaking that took the argument and didn't want to break it. Maybe we don't need it anymore.